frontend/public/reducers/observe.ts

```diff
@@ -3,12 +3,12 @@ import { List as ImmutableList, Map as ImmutableMap } from 'immutable';
 import { Alert, AlertStates, RuleStates, SilenceStates } from '@console/dynamic-plugin-sdk';
 
 import { ActionType, ObserveAction } from '../actions/observe';
-import {
-  MONITORING_DASHBOARDS_DEFAULT_TIMESPAN,
-  MONITORING_DASHBOARDS_VARIABLE_ALL_OPTION_KEY,
-} from '../components/monitoring/dashboards/types';
 import { isSilenced } from '../components/monitoring/utils';
 
+export const MONITORING_DASHBOARDS_DEFAULT_TIMESPAN = 30 * 60 * 1000;
+
+export const MONITORING_DASHBOARDS_VARIABLE_ALL_OPTION_KEY = 'ALL_OPTION_KEY';
```

Comment:
```suggestion
const MONITORING_DASHBOARDS_DEFAULT_TIMESPAN = 30 * 60 * 1000;

const MONITORING_DASHBOARDS_VARIABLE_ALL_OPTION_KEY = 'ALL_OPTION_KEY';
```

It looks like these constants are now only used in this file.

Moreover, I think the various `ActionType.DashboardsPatchVariable*` Redux actions in this file can probably be removed as part of this change? But maybe you prefer to leave that for a separate PR?

Reply:
Thnx! I removed the `export` in the latest commit and create a separate issue to remove the redux actions [OU-555](https://issues.redhat.com/browse/OU-555).